install_wxpdfdoc.sh

```diff
@@ -98,10 +98,16 @@ config_options="
   --with-wx-prefix=$prefix
   --with-wx-exec-prefix=$exec_prefix
   --build=$build_type
-  --host=$LMI_TRIPLET
   --disable-dependency-tracking
 "
 
+# Provide the host only when cross-compiling.
```

Comment:
Thinking more about this, shouldn't we do this when `$LMI_TRIPLET != $build_type"? This is the definition of cross-compiling and would work even when targetting some other platform.

Also, I think we should omit both `--build` and `--host` when not cross-compiling, specifying `--build` is harmless, but useless and unusual.

Reply:
Fixed.